android/build.gradle

```diff
@@ -149,8 +152,13 @@ subprojects {
                     "-Xjvm-default=all"
             ]
             if (project.name in moduleFriends.keySet()) {
-                def outputJarPath = "build/intermediates/compile_library_classes/release/classes.jar"
-                extraCompilerArgs.add("-Xfriend-paths=${project.rootDir}/libraries/${moduleFriends[project.name]}/$outputJarPath")
+                def friendName = moduleFriends[project.name]
```

Comment:
This would be a small and handy Gradle plugin we could provide to use on the modules themselves.

Reply:
i thought about this, but its very undocumented and prone to breakage over different versions of AGP, Gradle, and Kotlin so I don't think its worth supporting right now (created #436 to track the enhancement if others are interested)